app/Http/Controllers/ChannelController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Suyabay\Http\Controllers;
+
+use Suyabay\Channel;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Suyabay\Http\Controllers\Controller;
+
+class ChannelController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $channels = Channel::orderBy('id', 'asc')->paginate(10);
+
+        return view('dashboard.pages.view_channels', compact('channels'));
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function createIndex()
+    {
+        return view('dashboard.pages.create_channel');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $channel = Channel::create([
+            'channel_name'         => $request->name,
+            'channel_description'  => $request->description,
+            'subscription_count'   => 0
+        ]);
+        if ($channel)
```

Comment:
There should be space between line 47 and 48 and also space before your final `return`